airflow/models/xcom.py

```diff
@@ -229,3 +230,20 @@ def deserialize_value(result) -> Any:
                       "for XCOM, then you need to enable pickle "
                       "support for XCOM in your airflow config.")
             raise
+
+
+def resolve_xcom_backend():
+    """Resolves custom XCom class"""
+    path_to_custom_class = conf.get("core", "xcom_backend")
```

Comment:
Can you use conf.getimport here? This provides much more user-friendly messages.

Reply:
Hm, where can I find information about if?

Reply:
https://github.com/apache/airflow/pull/8463

Reply:
Nice! Rebased and added this function.